docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -20,6 +20,14 @@ Please refer to the appropriate section depending on your context.
 
 GraalVM imposes a number of constraints and making your application a native executable might require a few tweaks.
 
+=== Quarkus Annotations
```

Comment:
Let's mark new features as `Preview` in both docs and APIs/Annotations until we are happy we will support them for the while 1.x series.

Reply:
Once we have decided on the API I don't think there is any issue supporting these, they are just a new facade on functionality we already have.